### PR TITLE
97 editing contract data from admin page

### LINKED ISSUE #97: QoL: Editing contract data from admin page
## Description
As a contract manager i want to edit already existing contracts so that i do not have to delete and create a new contract every time i want to make changes.

## Acceptance criteria
- [x] There is an edit button on the admin page that makes it possible to manipulate the current data.
- [x] The fields in already existing contracts can be edited
- [x] Once a contract is edited, the database should store the new contract data.
- [x] The tabs on the admin pages are more clearly visible

### PR DESCRIPTION
**Pull request by martinjonsson01**
## Summary
Makes contracts editable from the admin page.

The new tabs:
chrome_o7pkdoLdZy

Example of changing a contract name:
chrome_dIPldKbZQ5

Example of changing a contract logo:
chrome_d8EvlbHBc1

## Definition of done
- [x] All acceptance criteria are completed.
- [x] The code has (at least) 80% branch test coverage.
- [x] All tests pass.
- [ ] The code is reviewed and accepted by (at least) two other team members.
- [x] Views adhere to design style.

Closes #97

### CODE REVIEW

**Review by Matte02 — APPROVED** [review]
Looks good, just have a question about the gifs. It looks like you are pressing way above the actual buttons, i.e in the first gif when swapping between the taps  I hope it's just the recording that is messed up and not like that in reality.

**Comment by martinjonsson01** [comment]
@Matte02 
> It looks like you are pressing way above the actual buttons, i.e in the first gif when swapping between the taps I hope it's just the recording that is messed up and not like that in reality.

Yeah, that's just the GIF. I think it's because of my Windows scaling being set to 125%, the recording software doesn't take that into account.

**Review by pkasinski — APPROVED** [review]
Looks good!

File: src/Client/Pages/Admin/ContractForm.razor [hunk 1]
```diff
@@ -6,8 +6,9 @@
 @using Blazorise.Markdown
 @inject ILogger<AdminPage> _logger
 @inject HttpClient _http
+@inject IJSRuntime _js
 
-<h4>Skapa nytt kontrakt</h4>
+<h4 id="contract-form-title" style="scroll-margin-top: 4.5rem;">Skapa nytt kontrakt</h4>
```
[the remark this answers is not in the document]
Comment (martinjonsson01) [reply src/Client/Pages/Admin/ContractForm.razor:11]: Good catch 👍

**Pull request merged**